### ISSUE TITLE
Custom release commands convert all args to atoms

### ISSUE THREAD

**Issue by doughsay** [bitwalker/distillery#537]
It appears that all custom release commands (using the `set commands:` option in `rel/config.exs`) will try and convert all arguments given to atoms.  I was trying to create a release task that would take an arbitrarily long string (representing space separated ids) as input.  This results in:

```
./bin/my_app my_custom_command "very long string here"
```

```
** (SystemLimitError) a system limit has been reached
    :erlang.binary_to_atom("very long string here")
```

### Steps to reproduce

1. Create a custom release task that takes one argument
2. Call the task from the release and pass a very long string

### Verbose Logs

How do I use `--verbose` with custom commands?

### Description of issue

- Expected results: I would expect my custom user input is not converted to atoms.
- What version of Distillery?: `2.0.5` and `2.0.9`
- Docker base image: `bitwalker/alpine-elixir:1.7.2`
- my `rel/config.exs` contains:

```
set commands: [
  my_custom_command: "rel/commands/my_custom_command.sh"
]
```

and the shell script is:

```
#!/bin/sh

release_ctl eval --mfa "MyApp.ReleaseTasks.my_custom_command/1" -- "$@"
```

### Additional Info

I also tried bypassing the custom shell script and calling the `eval` command in my release manually:

```
./bin/my_app eval "MyApp.ReleaseTasks.my_custom_command(\"long string here\")"
```

And I get a similar error, where it appears to be trying to convert the entire eval'ed code to an atom:

```
** (SystemLimitError) a system limit has been reached
    :erlang.binary_to_atom("MyApp.ReleaseTasks.my_custom_command(\"long string here\")")
```

**Comment by bitwalker** [comment]
I believe there were some issues with that version of 2.x, could you update to the latest 2.0.x release and try again? I'm not able to reproduce with master, but I haven't yet tried 2.0.9 to double check, though I have some tests against it there.

**Comment by doughsay** [comment]
Thanks for the quick response!  Sorry I didn't try with the latest version to begin with...  I just retried the exact same steps from above using the latest version `2.0.9` and it still produces the same results.  I'm going to edit the issue to include version `2.0.9` to reflect this.

**Comment by bitwalker** [comment]
This should be fixed in master/2.0.10.
